ioservice/io_foms.c

```diff
@@ -2037,6 +2057,42 @@ static int stob_io_create(struct m0_fom *fom)
 		return M0_ERR(-ENOMEM);
 
 	rwfop = io_rw_get(fom->fo_fop);
+	rw_replyfop = io_rw_rep_get(fom->fo_rep_fop);
+	/** TODO: Check this and use function call */
+	unit_size = (m0_lid_to_unit_map[rwfop->crw_lid]) >>
+			m0_stob_block_shift(fom_obj->fcrw_stob);
+	if ((m0_is_read_fop(fom->fo_fop)) && rwfop->crw_cksum_size ) {
+		/* For read this is not used so making it NULL */
+		//M0_ASSERT(rwfop->crw_di_data_cksum.b_nob == 0);
+		//M0_ASSERT(rwfop->crw_di_data_cksum.b_addr == NULL);
+
+		/* Init tracker variable, this gets updated in stobio_complete_cb */
+		rw_replyfop->rwr_cksum_nob_read = 0;
```

Comment:
@yeshpal-jain-seagate - Should we remove this tracker now ?

Reply:
Good for debugging so we can keep it